inst/pages/30_differential_abundance.qmd

```diff
@@ -5,78 +5,48 @@ library(rebook)
-data entail reference taxa approach used by DACOMP and RAIDA, analyzing the
-pattern of pairwise log ratios as done by ANCOM and bias-correction applied by
-ANCOMBC.
-
-We recommend to have a look at @Nearing2022. In this study, multiple DAA methods
-were applied to 38 different datasets and their results were compared to one
-another. Because each method follows a slightly different approach in terms of
-assumptions and normalization techniques, it was shown that results on the same
-dataset may differ substantially depending on the method. Recently, @Yang2022
-comprehensively evaluated DAA methods via a semi-parametric framework and 106
-real datasets. This study also concluded that different methods can produce
-contradictory results, creating the risk of cherry-picking the most favorable
-options for one’s own hypothesis. Therefore, it is highly recommended to perform
-DAA with multiple methods to determine whether the findings can be reproduced by
-different approaches. Built on the findings of @Calgaro2020, the *benchdamic* [@Calgaro2022] package could offer a valuable support in this regard. Through a comprehensive evaluation process it serves both practitioners by comparing DA methods from existing literature, and method developers by providing an impartial tool to evaluate their new approaches in comparison to what is already available. For details, check its extensive [vignette](https://www.bioconductor.org/packages/release/bioc/vignettes/benchdamic/inst/doc/intro.html).
+Microbiome data display unique characteristics that are exclusively addressed by DAA tools developed for microbiome analysis. These characteristics include high variability, zero-inflation and compositionality. *High variability* expresses that abundance of taxa often can vary significantly  from sample to sample. *Zero-inflation* means that typically more than 70% of the values are zeros, which could be either due to physical absence (structural zeros) or insufficient sampling effort (sampling zeros). *Compositionality* means that a change in the absolute abundance of one taxon will lead to apparent variations in the relative abundances of other taxa in the same sample. If neglected, such properties may cause significant bias in the results of DAA. Therefore, several approaches have been developed to address the unique properties of microbiome data and provide statistically reliable results.
+
+  ##  Approaches to Handle Zero-Inflated Data : 
+
+      The first approach to target zero-inflated data consists of specialized models, such as       *over-dispersed count models* and *zero-inflated mixture models*. DESeq2, edgeR and           corncorb are based on over-dispersed count models, whereas metagenomeSeq, RAIDA, ZIBB         and Omnibus implement zero-inflated mixture models to address zero-inflation.                 Typically, these models assume a negative binomial, beta-binomial or normal/log-normal        distribution.
```

Comment:
some strange extra spaces here?

Reply:
also elsewhere

Reply:
Yeah I tried making the indentation look neater but the formatting is somehow not consistent, working on it.

Reply:
I guess sometimes the line lengths are cut to max 80 chars per line, sometimes not.

Reply:
I sse. I had one query, OMA book has titles with numbered headings however the code didn't have it, why is it so? Should I add the numbered headings manually in the code?

Reply:
The numbering is automatic. Do not add your own numbers.